doc/appendix-at-rksv/cash-register-integration/cash-register-integration.md

```diff
@@ -92,15 +92,15 @@ Once the queue has been closed with a stop receipt, no hashing and signing of re
 
 #### End of Failure Receipt (Collective Failure Report)
 
-If for technical reasons signatures cannot be created by the fiskaltrust.SecurityMechanism, receipts need to be issued (according to the RKSV) and marked with a comment "security mechanism failed". Once the technical failure has been resolved, a signed collective receipt must be issued to make up for the signature linking of all receipts issued during the technical failure.
+If for technical reasons signatures cannot be created by fiskaltrust.SecurityMechanisms, receipts need to be issued (according to the RKSV) and marked with a comment "security mechanism failed". Once the technical failure has been resolved, a signed collective receipt must be issued to make up for the signature linking of all receipts issued during the technical failure.
```

Comment:
If, for technical reasons, signatures cannot be..

Reply:
done